modules/dns/dns.rb

```diff
@@ -1,20 +1 @@
 require 'dns/dns_plugin'
-
-module Proxy::Dns
```

Comment:
If we're going to break the API, I wonder if we shouldn't do some bigger refactoring. I'd very much like https://github.com/ekohl/smart_proxy_dns_powerdns/blob/bd5f16c189fab2f4f3e03c7fb08ac8f893484ff9/lib/smart_proxy_dns_powerdns/dns_powerdns_main.rb#L36-L48 to be moved into the core.